radio/src/bluetooth.cpp

```diff
@@ -64,6 +64,7 @@ char * bluetoothReadline()
 
   while (1) {
     if (!btRxFifo.pop(byte)) {
+      TRACE("NO RESPONSE FROM BT MODULE");
```

Comment:
Given I cannot test with two radio connected, I made a change that should ease your concern